### PR TITLE
Reconcile Service Catalog in Application Namespaces

### PR DESCRIPTION
**Pull request by fbm3307**
This PR is for reconciling Service Catalog in Application Namespaces

### CODE REVIEW

File: test/acceptance/features/serviceCatalogForward.feature [hunk 1]
```diff
@@ -0,0 +1,115 @@
+@wip
```
Comment (filariow) [inline test/acceptance/features/serviceCatalogForward.feature:1]:
```suggestion
```

File: config/agents/app/rbac/role.yaml [hunk 2]
```diff
@@ -75,3 +75,12 @@ rules:
...
   - update
+- apiGroups:
+  - primaza.io
+  resources:
+  - servicecatalogs
+  verbs:
+  - get
+  - list
+  - update
+  - watch
```
Comment (filariow) [inline config/agents/app/rbac/role.yaml:86]: this should not be needed. As of now, the agent don't need permissions on ServiceCatalogs. Right?

```suggestion
```

**Review by filariow — CHANGES_REQUESTED** [review]
mostly good, a couple of nits

File: test/acceptance/features/serviceCatalogForward.feature (hunk 1 above)
Comment (fbm3307) [reply test/acceptance/features/serviceCatalogForward.feature]: done

File: config/agents/app/rbac/role.yaml (hunk 2 above)
Comment (fbm3307) [reply config/agents/app/rbac/role.yaml]: done

**Comment by fbm3307** [comment]
CLosing in favor of #85

**Pull request closed without merging**